src/mono/System.Private.CoreLib/src/System/Threading/Thread.Mono.cs

```diff
@@ -67,6 +66,9 @@ public partial class Thread
         private StartHelper? _startHelper;
         internal ExecutionContext? _executionContext;
         internal SynchronizationContext? _synchronizationContext;
+#if TARGET_UNIX
+        internal WaitSubsystem.ThreadWaitInfo WaitInfo;
```

Comment:
Nit: To follow the coding convention, this should be internal WaitSubsystem.ThreadWaitInfo WaitInfo => _waitInfo;